src/main/java/io/strimzi/kafka/quotas/StaticQuotaConfig.java

```diff
@@ -44,6 +46,8 @@ public class StaticQuotaConfig extends AbstractConfig {
     static final String STORAGE_CHECK_INTERVAL_PROP = CLIENT_QUOTA_CALLBACK_STATIC_PREFIX + ".storage.check-interval";
     static final String AVAILABLE_BYTES_PROP = CLIENT_QUOTA_CALLBACK_STATIC_PREFIX + ".storage.per.volume.limit.min.available.bytes";
     static final String AVAILABLE_RATIO_PROP = CLIENT_QUOTA_CALLBACK_STATIC_PREFIX + ".storage.per.volume.limit.min.available.ratio";
+    static final String FALLBACK_THROTTLE_FACTOR = CLIENT_QUOTA_CALLBACK_STATIC_PREFIX + ".throttle.factor.fallback";
```

Comment:
The other configs have the `_PROP` suffix, I think we should be consistent and have it here as well.

Reply:
Fixed in [dc3231c](https://github.com/strimzi/kafka-quotas-plugin/pull/37/commits/dc3231c5ade80c19e55752378eccf0d78b8a96ad)